src/view/components/countries/SelectedCountries.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from "react";
+import { CountryName } from "../../../data/constants/CountryEmissions";
+import { TileTooltip } from "../atomic/TileTooltip";
+
+type SelectedCountriesType = {
+    selectedCountries: Map<CountryName, number>;
+    removeSelectedCountry: (country: CountryName) => void;
+    setCountryPercentage: (country: CountryName, percentage: number) => void;
+};
+
+export const SelectedCountries = ({
+    selectedCountries,
+    removeSelectedCountry,
+    setCountryPercentage,
+}: SelectedCountriesType) => {
+    const [worldPercentage, setWorldPercentage] = React.useState(1);
```

Comment:
Should these React hooks be moved into a useSelectedCountries hook?

Reply:
Time to use contexts

Reply:
What's your advice on this? :)

Reply:
I'd say leave the one hook per screen for now - if there is some refactoring that needs to be done a component then that will be a trigger